README.md

```diff
@@ -44,21 +44,21 @@ In this workflow we build a new image with our playbook, setup secrets (private
 
 Relative path to playbook file relative to project source. This is mounted in the container at **/opt/app-root/src/PLAYBOOK_FILE**.
 
-**`INVENTORY_FILE`** (optional)
+**`INVENTORY_FILE`** (optional, but an inventory is required)
 
-Relative path to inventory file relative to project source. This is mounted in the container at **/opt/app-root/src/INVENTORY_FILE**.
+Path to the location of the inventory file within the image. It can be a relative path pointing to an inventory within the project's source; for example it can point to a dynamic inventory script or directory provided in the source.
 
-**`INVENTORY_URL`** (optional)
+**`INVENTORY_URL`** (optional, but an inventory is required)
 
-URL to inventory file. This is downloaded into the container as inventory file **/opt/app-root/src/inventory**.
+URL to inventory file. This is downloaded into the container.
 
-**`ALLOW_ANSIBLE_CONNECTION_LOCAL`** (optional)
+**`DYNAMIC_SCRIPT_URL`** (optional, but an inventory is required)
 
-If set to false all `ansible_connection=local` settings will be ignored.
+URL to dynamic inventory script. This is downloaded into the container. If the dynamic inventory script is python see **PYTHON_REQUIREMENTS**.
 
-**`DYNAMIC_SCRIPT_URL`** (optional)
+**`ALLOW_ANSIBLE_CONNECTION_LOCAL`** (optional)
 
-URL to dynamic inventory script. This is downloaded into the container as **/opt/app-root/src/dynamic_inventory_script**. If the dynamic inventory script is python see **PYTHON_REQUIREMENTS**.
+If set to *false* all `ansible_connection=local` settings in the inventory will be removed.
```

Comment:
This comes from #8 - see [Brenton's comment](https://github.com/aweiteka/playbook2image/pull/8#issuecomment-265568039). Basically: it's a way to "reuse" existing inventories that work with local connections when run from outside but can't work the same way from a container... I'll add that explanation there for more clarity.